maven-java/kalix-spring-boot-parent/pom.xml

```diff
@@ -122,14 +122,14 @@
                     </configuration>
                     <dependencies>
                         <dependency>
-                        <groupId>org.springframework.boot</groupId>
-                        <artifactId>spring-boot-maven-plugin</artifactId>
-                        <version>${spring-boot.version}</version>
+                            <groupId>org.springframework.boot</groupId>
+                            <artifactId>spring-boot-maven-plugin</artifactId>
+                            <version>${spring-boot.version}</version>
                         </dependency>
                     </dependencies>
                     <executions>
                         <execution>
-                            <phase>package</phase>
+                            <phase>repackage</phase>
```

Comment:
We sent a bunch of time because it would only work for the protobuf parent but not the spring based one. This was the trick that made it work. I've tested a deployed image on kalix and seems to work fine.